src/js/utils/__tests__/ErrorMessagesUtil-test.js

```diff
@@ -0,0 +1,176 @@
+const ErrorMessagesUtil = require('../ErrorMessagesUtil');
+
+describe('ErrorMessagesUtil', function () {
+
+  describe('#create', function () {
+
+    it('should correctly transpose the data', function () {
+      const config = ErrorMessagesUtil.create({
+        'foo': {
+          CONSTANT_1: 'message1',
+          CONSTANT_2: 'message2'
+        }
+      });
+
+      expect(config).toEqual({
+        CONSTANT_1: [
+          {regexp: /foo/, message: 'message1'}
+        ],
+        CONSTANT_2: [
+          {regexp: /foo/, message: 'message2'}
+        ]
+      });
+    });
+
+    it('should stack error messages in correct order', function () {
```

Comment:
As far as I know Javascript doesn't guarantee an order on Object. I wouldn't rely on the order.